tools/openocd/ft4232.cfg

```diff
@@ -0,0 +1,12 @@
+interface ftdi
+ftdi_vid_pid 0x0403 0x6011
+ftdi_channel 1
+ftdi_layout_init 0x0098 0x008b
+ftdi_tdo_sample_edge falling
+set ESP32_RTOS none
```

Comment:
No RTOS for debugging? Will the breakpoint messup the scheduler and watchdog?

Reply:
This config option is a bit weird.  It will be able to debug FreeRTOS as the symbols are there in the elf file. It is more for the openocd target to be aware of FreeRTOS but from what I read about it, at best it doesn't do much and at worse can cause the debugger to randomly detach...